website/default.html

```diff
@@ -0,0 +1,203 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

```

Comment:
What is this?

The commit adding this has “delete unused files” as its message, but it doesn’t actually delete anything. I’m confused!

Reply:
I meant to remove these files... not sure what is happening here